### DISCUSSION TITLE
How to desible Bombings and Air Raid Sirens on singleplayer?

### DISCUSSION THREAD

**Discussion by POTHAYO** [TEHE-Studios/ExpandedHelicopterEvents#351] [Q&A]
from the first day i keep geting bombed and sirens wont stop

**Comment by Chuckleberry-Finn** [comment]
If the sirens don't stop it's probably bugged, just reload your game.

To clarify, the "bombing" is just a sound, there isn't an event that bombs you.

The bombing sound and sirens event should only occur later, and not the first day. Did you change the settings?
